src/main/include/RobotContainer.h

```diff
@@ -15,9 +15,13 @@
 // I/O Subsystems
 #include "io/SwerveDriveIO.h"
 #include "subsystems/SwerveDrive.h"
```

Comment:
We should delete this SwerveDrive.h since it's declared below in the // Subsystems section